packages/iris-grid/src/sidebar/VisibilityOrderingBuilder.tsx

```diff
@@ -142,15 +155,12 @@ class VisibilityOrderingBuilder extends Component<
       lastSelectedColumns: [],
       searchFilter: '',
     });
-    onMovedColumnsChanged([]);
+    onMovedColumnsChanged(model.movedColumns);
+    model.setColumnHeaderGroups(model.layoutHints?.columnGroups);
```

Comment:
As mentioned elsewhere shouldn't need to call `setColumnHeaderGroups` from anywhere if you make it a `get`/`set` property and set it through `TableViewportUpdater`